test/integration-tests/shared/getPermitNFTSignature.ts

```diff
@@ -0,0 +1,58 @@
+import { BigNumberish, constants, Signature } from 'ethers'
+import { splitSignature } from 'ethers/lib/utils'
+import { INonfungiblePositionManager } from '../../../typechain'
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers'
+
+export default async function getPermitNFTSignature(
```

Comment:
did you copy this function from somewhere else? just wondering

Reply:
yes! i got it from here: https://github.com/Uniswap/v3-periphery/blob/697c2474757ea89fec12a4e6db16a574fe259610/test/shared/getPermitNFTSignature.ts